CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## Environment
```

Comment:
I can update both .md files once we decide to go with Turborepo & pnpm

Reply:
I think some parts like installation and usage are probably irrelevant for the whole monorepo and we should keep them in per-package readmes. This env description I wrote so far is mostly for someone who wants to actually add some code, which is why it made sense to me to put this in Contributing instructions. Feel free to rearrange though.

Maybe it would make sense to have multiple mds in `/docs/` separately for
1. environment & IDE setup
2. contributing a fix/feature in an existing package incl conventions and walkthrough (from fork through testing to generating a changeset and opening a PR)
3. realease flow and conventions
4. guide to add a package (though eventually it should be replaced with a template package once we establish common configurations for everything)